src/core/jpg.js

```diff
@@ -635,7 +657,13 @@ var JpegImage = (function JpegImageClosure() {
   }
 
   JpegImage.prototype = {
-    parse: function parse(data) {
+    parse(data, { dnlScanLines = null, } = {}) {
+      if (dnlScanLines !== null && !Number.isInteger(dnlScanLines) &&
+          !(dnlScanLines >= 0x0001 && dnlScanLines <= 0xFFFF)) {
+        warn('JpegImage.parse - invalid "dnlScanLines" parameter: ' +
+             dnlScanLines);
+        dnlScanLines = null;
+      }
```

Comment:
Since `dnlScanLines` is an internal parameter, and you already enforce that it is greater than 0, is it really necessary to strictly validate the value of `dnlScanLines` ?

Reply:
I suppose that we can do without this, I'll remove it.